docs/content/getting-started/index.md

```diff
@@ -89,22 +89,18 @@ This command:
 - Creates a port-forward from localhost to port 3000 inside the container so you can navigate to the app's frontend UI
 - Streams container logs to your terminal
 
-In your browser you should see the demo app:
+Once the sample application is running, you can access it by opening [http://localhost:3000](http://localhost:3000) in a browser:
 
 {{< image src="demo-screenshot.png" alt="Screenshot of the demo container" width=600px >}}
 <br /><br />
 
-Congrats! You're running your first Radius app. When you're ready to move on to the next step, use <kbd>CTRL</kbd>+ <kbd>C</kbd> to exit the command.
+Congrats, you're running your first Radius app! When you're ready to move on to the next step, use <kbd>CTRL</kbd>+ <kbd>C</kbd> to terminate the application.
 
 ## 5. Add Database
 
-This step will add a database (Redis Cache) to the application.
+This step will add a database (Redis Cache) to the application, you will:
 
-You can create a Redis Cache using [Recipes]({{< ref "guides/recipes/overview" >}}) provided by Radius. The Radius community provides Recipes for running commonly used application dependencies, including Redis.
-
-In this step you will:
-
-- Add Redis to the application using a Recipe.
+- Add Redis to the application using a [Recipe]({{< ref "guides/recipes/overview" >}}) .
```

Comment:
Feels like a situation where talking about Recipes is actually a bit distracting from the main purpose of the Getting Started. 

Linking to the concept here is enough I think?

Reply:
Agree on linking to Recipes being enough. Small update:

```suggestion
- Adding a Redis Cache to your app definition, leveraging [Recipes]({{< ref "guides/recipes/overview" >}}) for the underlying infrastructure.
```